### PR TITLE
Added LottieFiles support

### PR DESCRIPTION
**Pull request by Bosch-0**
# Description

Addresses issue https://github.com/ZeusLN/zeus/issues/982

Kept this PR small but I did try importing a Lottie file that is the new loader animation (https://github.com/ZeusLN/zeus/issues/952) which worked well, see below: 

https://user-images.githubusercontent.com/55287964/169474538-299b36d6-b940-4827-9aee-995a9693b60e.mp4

Will open a separate PR that converts all the old loaders to this new loader as shown in the video. 

This pull request is categorized as a:

- [x] New feature
- [ ] Bug fix
- [ ] Code refactor
- [ ] Configuration change
- [ ] Locales update
- [ ] Other

## Checklist
- [x] I’ve run `npm run tsc` and made sure my code compiles correctly
- [x] I’ve run `npm run lint` and made sure my code didn’t contain any problematic patterns
- [x] I’ve run `npm run prettier` and made sure my code is formatted correctly
- [x] I’ve run `npm run test` and made sure all of the tests pass

## Testing

If you modified or added a utility file, did you add new unit tests?

- [ ] No, I’m a fool
- [ ] Yes
- [x] N/A

I have tested this PR on the following platforms (please specify OS version and phone model/VM):

- [x] Android
- [ ] iOS

I have tested this PR with the following types of nodes (please specify node version and API version where appropriate):

- [x] LND
- [ ] c-lightning-REST
- [ ] spark
- [ ] Eclair
- [ ] LndHub

### Locales
- [ ] I’ve added new locale text that requires translations
- [ ] I’m aware that new translations should be made on the Zeus [Transfix page](https://www.transifex.com/ZeusLN/zeus/) and not directly to this repo

### Third Party Dependencies and Packages

- [x] Contributors will need to run `npm install` after this PR is merged in
- [x] 3rd party dependencies have been modified:
    * verify that `package.json` and `package-lock.json` have been properly updated
    * verify that dependencies are installed for both iOS and Android platforms

### Other:

- [ ] Changes were made that require an update to the README
- [ ] Changes were made that require an update to onboarding

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -54,6 +54,7 @@
     "js-sha256": "0.9.0",
     "jsc-android": "241213.1.0",
     "lodash": "4.17.21",
+    "lottie-react-native": "^5.1.3",
```
Comment (kaloudis) [inline package.json:57]: Remove the caret and run `npm i` again. This ensures that we the maintainers have to manually upgrade the dependency and we don't end up with an upgrade we don't want. This improves our security posture by having to assess each upgrade.

**Comment by kaloudis** [comment]
Animation seems a little quick.

Nonetheless, pumped you got it working!

File: package.json (hunk 1 above)
Comment (Bosch-0) [reply package.json]: Good point, fixed!

**Comment by Bosch-0** [comment]
Yeah will put it at a 2, 3 a bit fast

File: package-lock.json [hunk 2]
```diff
@@ -45,6 +45,7 @@
         "js-sha256": "0.9.0",
         "jsc-android": "241213.1.0",
         "lodash": "4.17.21",
+        "lottie-react-native": "^5.1.3",
```
Comment (kaloudis) [inline package-lock.json:48]: can you run `npm i` again?

**Comment by kaloudis** [comment]
Can you include the actual lottie files in this PR?

**Comment by kaloudis** [comment]
Closing in favor of https://github.com/ZeusLN/zeus/pulls

**Pull request closed without merging**